components/pedigree/resources/src/main/resources/pedigree/view/exportSelector.js

```diff
@@ -148,7 +193,11 @@ define([
 
             console.log("Export data: >>" + exportString + "<<");
             if (exportString != "") {
-                FileSaver.saveTextAs(exportString, fileName);
+                if (exportType != "image") {
+                    FileSaver.saveTextAs(exportString, fileName);
+                } else if (exportType == "image" && imageType == "svg") {
+                    FileSaver.saveTextAs(exportString, fileName, 'base64', 'data:image/png;base64');
```

Comment:
well, it is of course arguable what is bette, there is no clear better way.

To start with, it is there because for the generic library it may make sense, but for us file type is a very technical point that "I, the pedigree developer" couldn't care less about, I just want to "save the file", whatever way current technology/browsers support it. E.g. I want to expose as high a level API as reasonable, and since we generally dont deal with file here I think abstractin away low-level MIME details is reasonable. Of course it is ok to leave a sis since it works :), just saying my 2c